samples/csharp_dotnetcore/51.teams-messaging-extensions-action/Bots/TeamsMessagingExtensionsActionBot.cs

```diff
@@ -109,64 +154,328 @@ private MessagingExtensionActionResponse ShareMessageCommand(ITurnContext<IInvok
             };
         }
 
-        private class CreateCardData
+        private MessagingExtensionActionResponse SendWebViewResponse(ITurnContext<IInvokeActivity> turnContext, MessagingExtensionAction action)
+        {
+            // The user has chosen to create a card by choosing the 'Create Card' context menu command.
+            CreatThumbnailCardData cardData = JsonConvert.DeserializeObject<CreatThumbnailCardData>(action.Data.ToString());
+            var card = new ThumbnailCard
+            {
+                Title = "ID: " + cardData.EmpId,
+                Subtitle = "Name: " + cardData.EmpName,
+                Text = "E-Mail: " + cardData.EmpEmail,
+                Images = new List<CardImage> { new CardImage("https://3er1viui9wo30pkxh1v2nh4w-wpengine.netdna-ssl.com/wp-content/uploads/prod/2014/10/MSFT_logo_rgb_C-Gray-768x283.png") },
```

Comment:
This link redirects to https://news.microsoft.com/wp-content/uploads/prod/2014/10/MSFT_logo_rgb_C-Gray-768x283.png Please host this .png in the project, and reference via baseurl path.